Updates to BSSC catalog page

emthompson-usgs opened the pull request.

lisa-usgs commented:
Looks good, but one suggestion...
instead of hotlinking the URL for the Vs30 input grids, just hotlink the text "Vs30 input grids".

Is the "30" supposed to be in the <sub> tag?  I guess I'm just used to seeing it as Vs30, so that may indeed be correct.

emthompson-usgs commented:
I usually use a convention that subscripts the "S30". But it does seem like a bit too much of a hassle for web pages when Vs30 is just as well understood. So I've removed it. I just added the commit with these changes.
